docs/REST/authentication.md

```diff
@@ -21,7 +21,7 @@ Provide each request with all HTTP headers required for Authentication. The mini
 
 **Example:**
 ```
-Host: example.azconfig.io
+Host: {myconfig}.azconfig.io
```

Comment:
Since this is an example, maybe we should use contoso.azconfig.io? In line 95, there is another *example.azconfig.io* that we probably should keep consistent with.

Reply:
We did this in the docs because those names aren't reserved. Looks like line 95 wasn't missed.